site/content/en/references/kustomize/builtins/_index.md

```diff
@@ -757,6 +757,11 @@ Each entry can have following fields:
 - `configHome`: [Optional] The value that kustomize should pass to helm via
   `HELM_CONFIG_HOME` environment variable.
    If omitted, `{tmpDir}/helm` is used, where `{tmpDir}` is some temporary.
+- `skipHooks`: [Optional] Skips running hooks when inflating the chart. By default
+  `helm template`, which is used to inflate helm charts, will render all the contents
```

Comment:
I think the fact that we're literally running `helm template` is an implementation detail that we don't need to necessarily surface to the docs. How about we change this to:

```
- `skipHooks`: [Optional] Skips running hooks when inflating the chart

Many of the above fields are analogous to arguments that would be passed to the helm CLI's `helm template` command.
```

It's a slight distinction, but IMO an important one.

Reply:
@natasha41575 I've rephrased it so that it doesn't explicitly mention `helm template`, but I do think it's valuable to provide a little more detail here than just "Skip running hooks when inflating the chart" (for example, when I first ran into this problem I had no idea what a "hook" was or why I was ended up with failed pods in my deploy).